File: cache/cache-annotation-processor/src/test/java/ru/tinkoff/kora/cache/annotation/processor/testcache/DummyCache2.java
package ru.tinkoff.kora.cache.annotation.processor.testcache;

import ru.tinkoff.kora.cache.CacheKey;
import ru.tinkoff.kora.cache.LoadableCache;
import ru.tinkoff.kora.cache.annotation.Cache;
import ru.tinkoff.kora.cache.caffeine.CaffeineCache;

import java.math.BigDecimal;

@Cache("dummy")
public interface DummyCache2 extends CaffeineCache<CacheKey.Key2<String, BigDecimal>, String> {

}

File: cache/cache-symbol-processor/src/test/kotlin/ru/tinkoff/kora/cache/symbol/processor/testcache/DummyCache2.kt
package ru.tinkoff.kora.cache.symbol.processor.testcache

import ru.tinkoff.kora.cache.CacheKey
import ru.tinkoff.kora.cache.annotation.Cache
import ru.tinkoff.kora.cache.caffeine.CaffeineCache
import java.math.BigDecimal
import java.math.BigInteger

@Cache("dummy2")
interface DummyCache2 : CaffeineCache<CacheKey.Key2<String?, BigDecimal?>, String>
